.github/workflows/ci-sql-cli.yaml

```diff
@@ -99,6 +99,7 @@ jobs:
     env:
       AIRFLOW__CORE__XCOM_BACKEND: astro.custom_backend.astro_custom_backend.AstroCustomXcomBackend
       AIRFLOW__ASTRO_SDK__STORE_DATA_LOCAL_DEV: True
+      TERMINAL_WIDTH: 3000
```

Comment:
@feluelle do you see any downsides to increasing the terminal width to this value?

Reply:
Not really. The advantage is that we can start having tests checking for explicit new lines whereas if we don't set the terminal width we need to either replace the new lines always, but then we cannot check for new lines in output, to make it consistent or check for the new lines in conjunction with the actual string/output. The question is what would be ideal terminal width be? I think there is none, hence we are setting it to a large number to avoid terminal doing terminal things.

Reply:
Sounds like a good plan! I'm happy for us to merge this once the checks are passing!

Reply:
It seems setting `TERMINAL_WIDTH` does not change reflect the terminal size in the tests. I can reproduce on a small local terminal :(